sdk/storage/test-resources.json

```diff
@@ -0,0 +1,180 @@
+{
```

Comment:
Optional nit - the formatting on this file is pretty funky.  Might be worth auto-formatting in VS Code.

Reply:
I actually switched it to use tabs in the wake of that email thread, which I guess does not format nicely in Github.

Reply:
That email was just to start a conversation and I don't think we've made any collective decisions for the repo.  I'd follow existing conventions for the time being.

Reply:
I didn't think we had made a decision to switch everything to tabs, but since I was using tabs locally for this file, and it was a new non-cs file, I figured it would be okay to update.
I will switch back to spaces.